metaswitch/common/mib.py

```diff
@@ -59,6 +62,8 @@ def get_oids_at_depth(self, depth):
 class Statistic(object):
     ''' The class structure for each OID and its relevant information
     '''
+    def __repr__(self):
+        return self.details['SNMP NAME']

```

Comment:
It this the first thing someone wants to see when they look at this class? I'd be inclined to drop it to the bottom of the class definition.